### PR TITLE
Wide iframe setting for Blog Pages, Article Pages, and Mozfest Pages

### LINKED ISSUE #6714: Blog and Article Wide Iframe setting
### Describe the bug
Blog and Article Pages don't have a "wide iframe" setting. This is less of a bug, and more of a woops we forgot a thing. This needs to be done before the general streamfield refactor we're planning on doing soon. 

Original spike ticket can be found here: #6672 

#### Blog:
- [ ] Does not have a wide iframe setting. All iframes are "wide" by default. 

#### Article page:
- [ ] Does not have a wide iframe setting.

### PR DESCRIPTION
**Pull request by KalobTaulien**
Closes #6714 

**Note**: Added support for Mozfest pages as well because iframe_block.html is used on Mozfest Primary Pages.

**Link to sample test page**: 
**Article Page:** https://foundation-s-6714-wide--cx66hw.herokuapp.com/en/publication-page-with-child-article-pages/fixed-title-article-page/
**Blog Page**: https://foundation-s-6714-wide--cx66hw.herokuapp.com/en/blog/initial-test-blog-post-with-fixed-title/

### CODE REVIEW

**Comment by KalobTaulien** [comment]
Note: this template is getting very hard to maintain. We'll be solving this with a major stream block refactor soon - but we needed a quick and dirty solution to make this work.

**Comment by KalobTaulien** [comment]
@danielfmiranda I tagged you for review on this one. 
@Pomax I tagged you so you know what's going on while you work on the streamblock PoC

File: network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/iframe_block.html [hunk 1]
```diff
@@ -14,11 +14,15 @@
...
+  {# Break out of the article_page.html containment grid #}
   </div> <!-- end .offset-sm-1.col-sm-10.col-12.article-blocks -->
+{% else %}
+  {# Mozfest page. Break out of the containment grid #}
+    </div>
+  </div>
 {% endif %}
 
-  <div class="col-lg-12">
+  <div class="{% if self.wide_iframe %}col-lg-12{% else %}col-lg-8 offset-lg-2{% endif %} {% if page_type != 'blog' and page_type != 'article' %}p-0{% endif %}">
```
Comment (KalobTaulien) [inline network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/iframe_block.html:25]: Getting very unmaintable. We had at least three pages to consider:
* Mozfest primary page 
* Blog detail page
* Article page

**Review by danielfmiranda — APPROVED** [review]
Hi Kalob, 

Thanks for tagging me on this issue, as I think this exposure will help with the streamblock proof of concept ticket I have assigned this sprint. 

After giving the files a look, everything looks good to me, though I do agree with you that the breaking out of the grid then going back in might get more difficult in the future, though I do hope the completion of the streamfield work might help alleviate this. 

That said, everything looks good regarding this ticket, so I am approving! 👍

File: network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/iframe_block.html (hunk 1 above)
Comment (Pomax) [reply network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/iframe_block.html:25]: The streamfields-as-containers (rows) work hopefully makes this the last hotfix.

**Review by Pomax — APPROVED** [review]

**Pull request merged**